src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
```

Comment:
The use of `&__star` suggests that you are following the BEM naming convention. However, ensure that the parent class `.stars` is used consistently as a block name. If `stars` is the block, `stars__star` should be used instead of `&__star` to maintain clarity and consistency in BEM naming.